src/events/request/RequestEventHandler.ts

```diff
@@ -58,6 +65,32 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 			return;
 		}
 
+		if ( BotConfig.request.invalidRequestJql ) {
+			const tickets = Array.from( this.getTickets( this.replaceTicketReferencesWithRichLinks( origin.content, regex ) ) );
```

Comment:
What's the reason for the `replaceTicketReferencesWithRichLinks` here? Wouldn't it work without it too?

Reply:
Idk why I used that, it should work just fine without it